src/server/twitterSearch.js

```diff
@@ -23,43 +33,102 @@ const makeTweeterFromRaw = (raw) => {
     .build();
 };
 
-export const searchAndSave = (res, query) => {
-  T.get('search/tweets', { 'q': query, 'count': 300 }, function (err, result, response) {
-    let count = 0;
-    result.statuses.forEach((tweetRaw) => {
-      const userRaw = tweetRaw.user;
+/**
+ * Given a raw tweet, extract information about the tweeter,
+ * if it was a retweet, etc, and store all information in
+ * the database.
+ * @param db The OrientDB instance
+ * @param tweetRaw The original status object from the Twitter API
+ * @returns {Promise.<T>}
+ */
+const processTweet = (db, tweetRaw) => {
+  const tweeter = makeTweeterFromRaw(tweetRaw.user);
+  const retweetedStatusRaw = tweetRaw['retweeted_status'];

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [['retweeted_status'] is better written in dot notation.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2175855561/issues/source?bid=3148782&fileBranchId=3163167#l46)